<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>腐蚀电化学测试系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>961</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>实验前准备：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>焊接电极</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>配制环氧树脂</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>制备电极</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>配制电解液</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>连接三电极</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>开始实验</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>焊接电极</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>450</width>
           <height>450</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>390</width>
           <height>390</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>配制环氧树脂</string>
        </attribute>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>390</width>
           <height>390</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>390</width>
           <height>390</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>0</y>
           <width>390</width>
           <height>390</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>390</width>
           <height>390</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>制备电极</string>
        </attribute>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>250</y>
           <width>250</width>
           <height>250</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>250</y>
           <width>350</width>
           <height>250</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>250</width>
           <height>250</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>0</y>
           <width>350</width>
           <height>250</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>配制电解液</string>
        </attribute>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>390</width>
           <height>390</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>390</width>
           <height>390</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>0</y>
           <width>390</width>
           <height>390</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>390</width>
           <height>390</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>连接三电极</string>
        </attribute>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>450</width>
           <height>450</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>390</width>
           <height>390</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>稳态测试</string>
    </property>
    <addaction name="actOCP"/>
    <addaction name="actScan"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>交流阻抗测试</string>
    </property>
    <addaction name="actZsw"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actIniTest"/>
   <addaction name="actOCP"/>
   <addaction name="actZsw"/>
   <addaction name="actScan"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本系统</string>
   </property>
  </action>
  <action name="actDoc_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/122.bmp</normaloff>:/images/images/122.bmp</iconset>
   </property>
   <property name="text">
    <string>打开文档</string>
   </property>
   <property name="toolTip">
    <string>打开文档</string>
   </property>
  </action>
  <action name="actScan">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/scan.png</normaloff>:/images/images/scan.png</iconset>
   </property>
   <property name="text">
    <string>动电位扫描</string>
   </property>
   <property name="toolTip">
    <string>动电位扫描</string>
   </property>
  </action>
  <action name="actOCP">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/OCP.png</normaloff>:/images/images/OCP.png</iconset>
   </property>
   <property name="text">
    <string>开路电位</string>
   </property>
   <property name="toolTip">
    <string>开路电位</string>
   </property>
  </action>
  <action name="actZsw">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/zsw.png</normaloff>:/images/images/zsw.png</iconset>
   </property>
   <property name="text">
    <string>交流阻抗</string>
   </property>
   <property name="toolTip">
    <string>交流阻抗</string>
   </property>
  </action>
  <action name="actIniTest">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/100.bmp</normaloff>:/images/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>实验前准备</string>
   </property>
   <property name="toolTip">
    <string>实验前准备</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
